ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -767,10 +767,8 @@ def main(argv):
     verbose = False
 
     try:
-        opts, args = getopt.getopt(argv, "a:r:s:l:o:m:i:e:vh",
-                                   ["action=", "run_id=", "start_marker=", "logs=",
-                                    "out_dir=", "match_files_in=", "ignore_files_in=",
-                                    "expect_files_in=", "verbose", "help"])
+        opts, args = getopt.getopt(argv, "a:r:s:l:o:m:i:e:vh", [
+                                   "action=", "run_id=", "start_marker=", "logs=", "out_dir=", "match_files_in=", "ignore_files_in=", "expect_files_in=", "verbose", "help"])
```

Comment:
@mannytaheri Can you update this here -- the line is too long and build fails with the following error

ansible/roles/test/files/tools/loganalyzer/loganalyzer.py:771:121: E501 line too long (173 > 120 characters)